config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
what does this do and do we need it downstream for OpenShift 4.12+ ?

Reply:
This is just for `kustomize` - makes it more "correct" (it's autogenerated). This doesn't translate into kubernetes manifests in any means.